Dockerfile

```diff
@@ -42,6 +42,8 @@ RUN helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx && \
 # Leonardo will install the chart from local version.
 RUN pushd /leonardo && \
     helm pull terra-app-setup-charts/terra-app-setup --untar && \
+    helm pull galaxy/galaxykubeman --untar && \
```

Comment:
Seeing a lot of this in tests:
```
 failed to download \"galaxy/galaxykubeman\" (hint: running `helm repo update` may help)
```

Thinking it might be a good idea to cache the charts we use anyway on disk instead of downloading it each time.